deepspeed/inference/engine.py

```diff
@@ -324,7 +324,7 @@ def _validate_args(self, mpu, replace_with_kernel_inject):
         if self._config.checkpoint is not None and not isinstance(self._config.checkpoint, (str, dict)):
             raise ValueError(f"checkpoint must be None, str or dict, got {type(self._config.checkpoint)}")
 
-        supported_dtypes = [None, torch.half, torch.int8, torch.float]
+        supported_dtypes = [None, torch.half, torch.int8, torch.float, torch.bfloat16]
```

Comment:
Do you need to adjust this as well?
https://github.com/microsoft/DeepSpeed/blob/4e74b8d066b27d34a0061d62dede2ce4b4697d1e/deepspeed/inference/engine.py#L76-L77

Reply:
@tjruwase I can generalize it, so int8 would also be checked, via accelerate.supported_dtype() method.
but i see that no accelerator support int8.

Reply:
@tjruwase  Done